src/fable-library-py/fable_library/date.py

```diff
@@ -224,9 +260,26 @@ def op_addition(x: datetime, y: timedelta) -> datetime:
 
 
 def parse(string: str, detectUTC: bool = False) -> datetime:
-    from dateutil import parser
```

Comment:
Also remove `python-dateutil` from `pyproject.toml` and do a `poetry update` to update `poetry.lock` remove the dependency